controllers/vrg_volrep.go

```diff
@@ -1889,7 +1892,13 @@ func (v *VRGInstance) restorePVsAndPVCsFromS3(result *ctrl.Result) (int, error)
 		// Ignoring PVC restore errors helps with the upgrade from ODF-4.12.x to 4.13
 		pvcCount, err = v.restorePVCsFromObjectStore(objectStore, s3ProfileName)
 
-		if err != nil || pvCount != pvcCount {
+		if err != nil {
+			v.log.Error(err, fmt.Sprintf("error fetching PVC cluster data from S3 profile %s", s3ProfileName))
+
+			continue
+		}
+
+		if pvCount != pvcCount {
 			v.log.Info(fmt.Sprintf("Warning: Mismatch in PV/PVC count %d/%d (%v)",
 				pvCount, pvcCount, err))
```

Comment:
We can use the same style to do WARNING level messages for pv/pvc restore failures - this is the same use case - we try to use one profile and something failed, so we move to next profile.

Using log.Error() we get a traceback and I'm not sure it is helpful when expected errors and the error message is not enough.

Reply:
Done